### PR TITLE
fix(usa): change adress labels

### PR DESCRIPTION
**Pull request by gsasouza**
#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [X] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
✔️  | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: packages/usa/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1] 2021-05-20
+- Change address label
```
Comment (lucasecdb) [inline packages/usa/CHANGELOG.md]:
```suggestion
### Changed
- Updates address's `street` and `complement` labels.
```

**Review by lucasecdb — APPROVED** [review]

**Comment by wdsrocha** [comment]
The code LGTM, but why this change is necessary?

**Comment by gsasouza** [comment]
because it's breaking [PlaceComponents](https://github.com/vtex-apps/place-components). `addressLine1` and `addressLine2` is not defined [here](https://github.com/vtex-apps/place-components/blob/91d26d68ea9dcf787274fd390b6678a2781c5af4/react/AddressForm.tsx#L15) and because of that it breaks when called [here](https://github.com/vtex-apps/place-components/blob/91d26d68ea9dcf787274fd390b6678a2781c5af4/react/AddressForm.tsx#L226)

**Review by wdsrocha — APPROVED** [review]
fair enough!

**Pull request merged**